src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx

```diff
@@ -272,7 +272,9 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
 
   onUpdate = () => {
     jsYaml.safeLoadAll(this.state.yamlModified, (objectModified: object) => {
-      const jsonPatch = JSON.stringify(mergeJsonPatch(objectModified, getIstioObject(this.state.istioObjectDetails)));
+      const jsonPatch = JSON.stringify(
+        mergeJsonPatch(objectModified, getIstioObject(this.state.istioObjectDetails))
+      ).replace(new RegExp(',null', 'g'), '');
```

Comment:
I don't think this will work in all cases.
https://datatracker.ietf.org/doc/html/rfc7386

"null" values are allowed in the merge/patch syntax and differentiate between an explicit field with "null" or removing a field from an object.

But it seems the problem happens with the "arrays" values, so this global regex will produce side effects when you want to remove a field in other editions.

Reply:
yes, it should apply only arrays, as in json, null  values in array are moved on the end, ',null/g' will replace them all, and it seems the original issue existed always

Reply:
changed the regexp to '"(,null)+]' to make sure only null in arrays are considered